app/presenters/digest_presenter.rb

```diff
@@ -1,6 +1,9 @@
 class DigestPresenter < NotificationPresenter
   attr_reader :current_project, :notifications, :template
 
+  # Jul 9th at 2:45pm
+  Time::DATE_FORMATS[:digest_format] = ->(time) { time.strftime "%h #{time.day.ordinalize} at %l:%M%P" }
```

Comment:
@brianp Is it possible to add timezone? I'm not too sure how to use the user's timezone, but maybe we can use UTC as default?

Reply:
It's a good idea, but we don't really do that anywhere else. We use the local_time gem which does some transformations on the front-end for timestamps. I think for now it's safe for everyone to assume it's the local system time of the VM without distinctly clarifying. Once we begin to offer a timezone setting for notifications etc it'll matter a little more.